Assignment4/Assignment4.js

```diff
@@ -0,0 +1,24 @@
+var fs = require('fs');
+console.log("------------------------------------------");
+function ReadAppendWrite (read_file, append_file, write_file) {
+  fs.readFile (read_file, function(err1, data1) {
+    if (err1) throw err1;
+    console.log(data1.toString())
+    console.log("----------------------------------------------");
+
+    fs.appendFile(append_file, data1, function(err2) {
+      if (err2)  throw err2;
```

Comment:
two spaces between if() and throw clause. One space required. Correct rest of the places as well